server/api/upload.go

```diff
@@ -36,9 +39,34 @@ func (a *API) HTTPUpload(c *gin.Context) {
 		c.Error(err)
 		return
 	}
+
+	go a.index(ctx, currentSnapshot, data)
```

Comment:
If we move our inmemory store to storage this step is not needed.

Reply:
Done